tools/Status/Status.py

```diff
@@ -114,10 +114,14 @@ def get_input_file(comment: Optional[str], work_dir: str) -> Optional[str]:
             )
     # Fallback: Get the input file from the submit context file if present
     context_file = Path(work_dir) / "SubmitContext.yaml"
-    if context_file.exists():
-        with open(context_file, "r") as open_context_file:
-            input_file = yaml.safe_load(open_context_file)["input_file"]
-            return os.path.join(work_dir, input_file)
+    try:
+        if context_file.exists():
+            with open(context_file, "r") as open_context_file:
+                input_file = yaml.safe_load(open_context_file)["input_file"]
+                return os.path.join(work_dir, input_file)
+    except PermissionError:
```

Comment:
Why only a `PermissionError`? Why not just catch any failure to read the context file?